pkg/model/estimator/local/ratio_vm.go

```diff
@@ -0,0 +1,68 @@
+/*
+Copyright 2023.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/*
+ratio_vm.go
+calculate virtual machine component and other power by ratio approach when node power is available.
+*/
+
+package local
+
+import (
+	"sync"
+
+	"k8s.io/klog/v2"
+
+	collector_metric "github.com/sustainable-computing-io/kepler/pkg/collector/metric"
+)
+
+func getVMResUsage(vm *collector_metric.VMMetrics, usageMetric string) float64 {
```

Comment:
since the VM power comes from the qemu process, would it make sense to create a <pid, vm> metrics mapping and reuse the process power metrics, instead of calculating VM power from scratch?

Reply:
I've now simplified the VM metrics by simply grab the Process metrics values that are already processed. 
So this is not needed anymore.

Reply:
this new approach is great!